tools/wptrunner/wptrunner/browsers/webkitgtk_minibrowser.py

```diff
@@ -0,0 +1,74 @@
+from .base import get_timeout_multiplier   # noqa: F401
+from .webkit import WebKitBrowser
+from ..executors import executor_kwargs as base_executor_kwargs
+from ..executors.executorwebdriver import (WebDriverTestharnessExecutor,  # noqa: F401
+                                           WebDriverRefTestExecutor)  # noqa: F401
+from ..executors.executorwebkit import WebKitDriverWdspecExecutor  # noqa: F401
+
+__wptrunner__ = {"product": "webkitgtk_minibrowser",
+                 "check_args": "check_args",
+                 "browser": "WebKitGTKMiniBrowser",
+                 "browser_kwargs": "browser_kwargs",
+                 "executor": {"testharness": "WebDriverTestharnessExecutor",
+                              "reftest": "WebDriverRefTestExecutor",
+                              "wdspec": "WebKitDriverWdspecExecutor"},
+                 "executor_kwargs": "executor_kwargs",
+                 "env_extras": "env_extras",
+                 "env_options": "env_options",
+                 "run_info_extras": "run_info_extras",
+                 "timeout_multiplier": "get_timeout_multiplier"}
+
+
+def check_args(**kwargs):
+    pass
+
+
+def browser_kwargs(test_type, run_info_data, config, **kwargs):
+    return {"binary": kwargs["binary"],
+            "webdriver_binary": kwargs["webdriver_binary"],
+            "webdriver_args": kwargs.get("webdriver_args")}
+
+
+def capabilities(server_config, **kwargs):
+    args = kwargs.get("binary_args", [])
+    if "--automation" not in args:
+        args.append("--automation")
+
+    return {
+        "browserName": "MiniBrowser",
+        "browserVersion": "2.20",
+        "platformName": "ANY",
```

Comment:
Is `platformName` required?

Reply:
`platforName` acts as a filter. The webdriver would refuse to start the browser if the platform it reports (would be: linux) doesn't match was asked. "any" acts as a wildcard. I guess I can remove this as that would have the same effect than specifying "any" here.

Reply:
Gone in https://github.com/web-platform-tests/wpt/pull/19227/commits/462c2dfbf7534d944af4fda1081018f8f6b21f0c.